ledger/src/blockstore_db.rs

```diff
@@ -227,6 +259,18 @@ impl Rocks {
         if let Some(recovery_mode) = recovery_mode {
             db_options.set_wal_recovery_mode(recovery_mode.into());
         }
+        if let Some(compression_mode) = compression_mode {
+            let mode = compression_mode.into();
+            let compression = [
+                DBCompressionType::None,
+                DBCompressionType::None,
+                mode,
+                mode,
+                mode,
+                mode,
+            ];
+            db_options.set_compression_per_level(&compression);
```

Comment:
@carllin @ryoqun I tried `set_compression_per_level` and also `db_options.set_compression_type`. Neither of them actually seem to work when I check the `rocksdb/LOG` file. When I add the `snappy` feature, then it uses `snappy` as the enabled compression option regardless of the setting here. Does it seem right?

Reply:
hmmmm because each column family has its own set of levels, I think you may have to add the `set_compression_per_level` to  the column family specific options returned from `get_cf_options`

Reply:
ah, that was it. Also had to read the log correctly. First it lists a "default" column family options, then each individually. I thought it was like a global setting.

Reply:
nice, glad it was a simple fix!